setup.py

```diff
@@ -41,9 +44,9 @@
         "Framework :: Wagtail",
```

Comment:
**chore:** did not note this in my previous review: but since we don't support Django 3.0 and 3.1 anymore, these classifiers should be removed.

Reply:
✅